.changeset/grumpy-socks-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/next': patch
+---
+
+Support incremental partial prerendering
```

Comment:
Did we agree on calling it "incremental adoption of partial prerendering"? Either way, the wording should be consistent.